scenarioo-client/app/shared/navigation/title.component.ts

```diff
@@ -0,0 +1,22 @@
+import * as angular from 'angular';
+import {ConfigurationService} from '../../services/configuration.service';
+import {Title} from '@angular/platform-browser';
+import {Component, Directive} from '@angular/core';
+import {downgradeComponent} from '@angular/upgrade/static';
+
+@Component({
+    selector: 'app-title',
+    template: '',
+})
```

Comment:
Have you considered using a service? (Since there is no actual HTML involved)

Reply:
don't you like having it on the index.html?